akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -340,7 +340,8 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `application/grpc+proto`                                                    = abin("grpc+proto", NotCompressible)
   val `application/lha`                                                           = abin("lha", NotCompressible, "lha")
   val `application/lzx`                                                           = abin("lzx", NotCompressible, "lzx")
-  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible, "pot", "pps", "ppt", "ppz")
+  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible, "pot", "pps", "ppz")
```

Comment:
And remove the ones here.

```suggestion
  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible)
```